content/applications/sales/sales/products_prices/prices/currencies.rst

```diff
@@ -1,49 +1,239 @@
+   The ability to instantly update the currency rates is also available, by clicking the
+   :guilabel:`circular arrows` icon, located to the right of the :guilabel:`Next Run` field.
 
-Automatic conversion from public price
-======================================
+   When all configurations are complete, click :guilabel:`Save` to save all changes.
 
-The public price is in your company's main currency (see
-:menuselection:`Accounting --> Settings`) and is set in product detail form.
+.. note::
+   All payment methods **must** be in the same currency as the Sales Journal, or the company
+   currency, if the company currency is not set. If not, a :guilabel:`Validation Error` message
+   appears.
 
-.. image:: currencies/public_price.png
+View, edit, and add currencies
+==============================
+
+To view, edit, and add currencies to the database, and make them available on pricelists and on the
+:guilabel:`Main Currency` drop-down, click the :guilabel:`Currencies` link, beneath the
+:guilabel:`Currency` field (on the *Accounting* app's :guilabel:`Settings` page).
+
+When the :guilabel:`Currencies` link is clicked, a separate :guilabel:`Currencies` page is revealed.
+
+.. image:: currencies/main-currencies-page.png
    :align: center
+   :alt: How the main currencies page appears in Odoo Accounting.
+
+On this page, Odoo provides a master list of 167 global currencies. Each row shows the corresponding
+:guilabel:`Currency`, :guilabel:`Symbol`, :guilabel:`Name`, date of the :guilabel:`Last Update`, and
+:guilabel:`Current Rate` (compared to the default currency of the country in which the company is
+based in).
 
-The conversion rates can be found in
-:menuselection:`Accounting --> Configuration --> Currencies`. They can be updated
-from Yahoo or the European Central Bank at your convenience: manually, daily, weekly,
-etc. See :menuselection:`Accounting --> Settings`.
+To the far right, there are two columns, which can be toggled on or off:
 
-.. image:: currencies/currency_rate.png
+- :guilabel:`Use on eBay`: meaning this currency can be used with the connected eBay account (if
+  applicable).
+- :guilabel:`Active`: meaning this currency is `activated`, which means it can be added to a
+  pricelist, or used as the :guilabel:`Main Currency` of the company, if desired (via
+  :menuselection:`Accounting app --> Configuration --> Settings --> Currencies section`).
+
+.. note::
+   By default, all the currency options that are :guilabel:`Active` are at the top of the list.
+
+.. tip::
+   It's recommended that *at least* one pricelist is created per :guilabel:`Active` currency. See
+   also: `/applications/sales/sales/products_prices/prices/pricing.html` to learn more about
+   pricelist configuration.
+
+To toggle these options on/off, click the toggle switch in the row for the corresponding column.
+When `on` the color of the switch is green. When `off`, the color of the switch is gray.
```

Comment:
I dont think literal formatting applies here, the user isnt inputting "on" or "off" but rather toggling the state of the icon similar to _folding_ a kanban column
```suggestion
When *on* the color of the switch is green. When *off*, the color of the switch is gray.
```